src/aura/contactList/contactList.css

```diff
@@ -1,6 +1,7 @@
-.THIS.custom-card{
-    display:inline-block !important;
-    max-height: 160px;
-    max-width: 350px;
+.THIS .slds-card {
     margin: 10px;
```

Comment:
<img width="1372" alt="screen shot 2017-10-26 at 2 07 34 pm" src="https://user-images.githubusercontent.com/32850485/32066726-16bf12d6-ba57-11e7-82d6-066563cf6c51.png">
this is the margin issue I'm speaking about, I'lll approve this changes but keep in mid that those margin left on the first items looks odd, that will happen also with the last items of that row